Source/WebInspectorUI/UserInterface/Controllers/NetworkManager.js

```diff
@@ -73,6 +73,18 @@ WI.NetworkManager = class NetworkManager extends WI.Object
                 for (let serializedLocalResourceOverride of serializedLocalResourceOverrides) {
                     let localResourceOverride = WI.LocalResourceOverride.fromJSON(serializedLocalResourceOverride);
 
+                    if (localResourceOverride.isRegex) {
+                        // Fix for stored local overrides created before URL regex checking was added. See <https://webkit.org/b/294072>
```

Comment:
we probably should have this be a new bug for eventually removing this as we ideally won't need this forever
```
// FIXME <https://webkit.org/b/######> Remove fix for stored local overrides created before URL regex checking was added
```
and you can have that bug be blocked by <https://webkit.org/b/294072>